instant-xml/src/lib.rs

```diff
@@ -150,7 +169,7 @@ impl<'xml, W: std::fmt::Write> Serializer<'xml, W> {
         Ok(())
     }
 
-    fn add_close_tag(&mut self, field_context: &FieldContext) -> Result<(), Error> {
+    fn add_close_tag(&mut self, field_context: &FieldContext) -> Result<()> {
```

Comment:
I know that some crates like to alias `Result`, but I'd much prefer not doing so, as I find it just makes it harder to reason about the code.